easyDiffractionApp/Logic/PyQmlProxy.py

```diff
@@ -1072,7 +1120,7 @@ def _setParametersAsObj(self):
                 "label": par_path,
                 "value": par.raw_value,
                 "unit": '{:~P}'.format(par.unit),
-                "error": par.error,
+                "error": float(par.error),
```

Comment:
If `par.error` can be `None`, this will throw. Should we be more careful here?

Reply:
`par.error` is an instance of the `uncertainties.core.CallableStdDev` class and cannot become `None`.